core/server/apps/amp/lib/helpers/amp_content.js

```diff
@@ -67,9 +68,28 @@ function ampContent() {
         };
 
     return Promise.props(amperizeHTML).then(function (result) {
+        var $;
+
+        // our Amperized HTML
         ampHTML = result.amperize || '';
 
-        // let's sanitize our HTML!!!
+        // Use cheerio to traverse through HTML and make little clean-ups
+        $ = cheerio.load(ampHTML);
+
+        // We have to remove source children in video, as source
+        // is whitelisted for audio, but causes validation
+        // errors in video, because video will be stripped out.
+        // @TODO: remove this, when Amperize support video transform
+        $('video').children('source').remove();
```

Comment:
We don't want to transform it. Only transforming the tag is not enough for AMP. That's why we use `amperize`.